index.html

```diff
@@ -1,14 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <link rel="stylesheet" href="src/css/style.css">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="src/css/style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Awesome Books</title>
-</head>
-<body>
-    <h1>hello my friend</h1>
+  </head>
+  <body>
+    <section id="list">
+      <h1>List</h1>
+      <ul id="storage_list"></ul>
+    </section>
+    <section id="add_book">
+      <input type="text" id="title" placeholder="title" required/>
+      <input type="text" id="author" placeholder="author" required/>
+      <button type="button" id="btn">Add</button>
```

Comment:
- [ ] When I click the add button on deployment it doesn't work, I kindly invite you to fix it. 🐶 
- Hint: I'm almost sure the problem is on the src script, check it out.
![image](https://user-images.githubusercontent.com/20563252/124510146-da2bc080-dd98-11eb-92dd-b8b7c6e5cd6e.png)